commons/src/main/java/org/infinispan/commons/util/SecurityActions.java

```diff
@@ -70,4 +75,62 @@ static String getProperty(String name) {
 
       return SysProps.PRIVILEGED.getProperty(name);
    }
+
+   private static <T> T doPrivileged(PrivilegedAction<T> action) {
+      if (System.getSecurityManager() != null) {
+         return AccessController.doPrivileged(action);
+      } else {
+         return action.run();
```

Comment:
@tristantarrant  OK, I updated it following the pattern in `org.infinispan.jmx.SecurityActions`. One thing to mention is that   it does not call `Security.doPrivileged(action)` here like what it does in `org.infinispan.jmx.SecurityActions`, because:

- `org.infinispan.security.Security` is in `core` module, this change is done in `commons` module, `core` depends on `commons`.
- Using `Security.doPrivileged(action)` will change the original way when there is no security manager enabled.